keyboards/splitish/keymaps/default/keymap.c

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2019 Reid Schneyer <reschneyer89@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+//Layer definitions
+#define BASE 0
+#define NUMS 1
+#define SYMB 2
+#define FNS  3
```

Comment:
```suggestion
enum layers {
  BASE,
  NUMS,
  SYMB,
  FNS
```
Better to use `enum` than a series of `#define`s here for a few reasons -- for instance they automatically index themselves, and you don't have to re-index manually if you insert a new layer down the line.